x-pack/plugins/secops/public/store/local/network/selectors.ts

```diff
@@ -15,6 +15,12 @@ import { GenericNetworkModel, NetworkType } from './model';
 const selectNetwork = (state: State, networkType: NetworkType): GenericNetworkModel =>
   get(networkType, state.local.network);
 
+export const dnsSelector = () =>
+  createSelector(
+    selectNetwork,
+    network => network.queries!.dns
+  );
+
```

Comment:
Is this selector right?

The others below it are optionally returning null.

```ts
export const networkFilterQueryExpression = () =>
  createSelector(
    selectNetwork,
    network => (network.filterQuery! ? network.filterQuery!.query.expression : null)
  );
```

Really, you don't need the `!` on that one below btw as well ;-). It can just be:

```ts
export const networkFilterQueryExpression = () =>
  createSelector(
    selectNetwork,
    network => (network.filterQuery ? network.filterQuery!.query.expression : null)
  );
```

And...Since we're on the topic of usage with these. I see that when they're detected as null they're just flipped around to empty string like this one usage in the code base:

```
filterQueryExpression: getNetworkFilterQuery(state, type) || '',
```

Wouldn't these then be better suited to creating a selector that returns an empty string if they're found to be null if they can be null and cut out the middle man?

Sorry for all the comments ... I haven't reviewed in a while and I'm mostly curious about ways to make the intent of the code easier to grok.